src/main/java/lotto/model/Money.java

```diff
@@ -1,27 +1,30 @@
 package lotto.model;
 
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
 import lotto.code.ErrorCode;
 import lotto.exception.LottoException;
 
 public class Money {
-	public static final int LOTTO_PRICE = 1000;
-	private static final double MATH_ROUND_VALUE = 100d;
+	public static final BigDecimal LOTTO_PRICE = new BigDecimal(1000);
```

Comment:
금액을 의미하는 상수도 Money 클래스로 래핑해보는건 어떨까요 :)

```suggestion
	public static final Money LOTTO_PRICE = new Money(new BigDecimal(1000));
```